.travis.yml

```diff
@@ -2,43 +2,59 @@ language: node_js
 node_js: 
   - "8"
   - "10"
+  - "12"
 
 script: npm run unit-test
 
+common: &integration-common
```

Comment:
Instead of common, I would call this `Defaults`.

I would put all other shared properties as subproperties of `Defaults`. This way there is only one property that travis can confuse for one of its special keywords.